components/org.wso2.carbon.identity.oauth.uma.xacml.service/src/test/java/org/wso2/carbon/identity/oauth/uma/xacml/service/handler/XACMLUMAHandlerTest.java

```diff
@@ -0,0 +1,9 @@
+package org.wso2.carbon.identity.oauth.uma.xacml.service.handler;
+
+
+/**
+ * Created by isuri on 5/30/18.
+ */
+public class XACMLUMAHandlerTest {
```

Comment:
Why we are having an empty test class without any methods.

Reply:
Also setup your IDE to include license and to remove default class level comment